t/35-script_clone_job.t

```diff
@@ -129,4 +130,60 @@ subtest 'asset download' => sub {
     ok(-f "$temp_assetdir/iso/bar.iso", 'foo touched');
 };
 
+subtest 'clone 2 nodes HA cluster with clone_job' => sub {
+    my %node1_settings = (
+        NAME          => '00007936-sle-15-SP3-Online-x86_64-Build67.1-sles4sap_hana_node01@64bit-sap',
+        TEST          => 'sles4sap_hana_node01',
+        HDD_1         => 'SLE-15-SP3-x86_64-Build67.1-sles4sap-gnome.qcow2',
+        HDDSIZEGB     => 60,
+        PARALLEL_WITH => 'sles4sap_hana_supportserver@64bit-2gbram',
+        NICTYPE       => 'tap',
+        WORKER_CLASS  => 'tap,qemu_x86_64',
+    );
+    my %node2_settings = (
+        NAME          => '00007935-sle-15-SP3-Online-x86_64-Build67.1-sles4sap_hana_node02@64bit-sap',
+        TEST          => 'sles4sap_hana_node02',
+        HDD_1         => 'SLE-15-SP3-x86_64-Build67.1-sles4sap-gnome.qcow2',
+        HDDSIZEGB     => 60,
+        PARALLEL_WITH => 'sles4sap_hana_supportserver@64bit-2gbram',
+        NICTYPE       => 'tap',
+        WORKER_CLASS  => 'tap,qemu_x86_64',
+    );
+    my %supportserver_settings = (
+        NAME         => "00007934-sle-15-SP3-Online-x86_64-Build67.1-sles4sap_hana_supportserver@64bit-2gbram",
+        TEST         => 'sles4sap_hana_supportserver',
+        HDD_1        => 'openqa_support_server_sles12sp3.x86_64.qcow2',
+        HDDSIZEGB    => 60,
+        NICTYPE      => 'tap',
+        WORKER_CLASS => 'tap,qemu_x86_64',
+    );
+    #my $jobid = 7934;
+    #clone_job($jobid, \%options);
```

Comment:
Please remove these leftovers.

Reply:
Yes of course, it's just a first push to be able to run the `.t` test, as I can't run it locally.

Reply:
To run the test locally, use `t/test_postgresql /dev/shm/tpg` and then `TEST_PG='DBI:Pg:dbname=openqa_test;host=/dev/shm/tpg' OPENQA_BASEDIR= prove -v t/35-script_clone_job.t` (from https://open.qa/docs/#testing). See https://github.com/Martchus/openQA-helper for an example of a good development environment.

Reply:
the documentation already shows a simpler example:

```
make test KEEP_DB=1 TESTS=t/35-script_clone_job.t
```

but your example also works showing the low level steps in detail.